### PR TITLE
release(v0.8.1): prepare release

### PR DESCRIPTION
**Pull request by smira**
This is the official v0.8.1 release.

Signed-off-by: Andrey Smirnov <smirnov.andrey@gmail.com>

### CODE REVIEW

**Review by andrewrynhard — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,11 +1,23 @@
 
+<a name="v0.8.1"></a>
+## [v0.8.1](https://github.com/talos-systems/talos/compare/v0.8.0...v0.8.1) (2020-12-29)
+
+### Fix
+
+* networkd updates for Packet, hostname detection, console kernel arg
```
Comment (andrewrynhard) [inline CHANGELOG.md:7]: Didn't we have 2 changes we wanted in this release?

**Review by andrewrynhard — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (smira) [reply CHANGELOG.md:7]: I even picked up three and had to add fourth one (to fight with the go-procfs changes), but as they're squashed... not sure what to do? three PRs? fix up change log manually?

Comment (smira) [reply CHANGELOG.md:7]: https://github.com/talos-systems/talos/commit/6eb8bf4d7a9a76238a6d181299e09f21c5612159

Comment (andrewrynhard) [reply CHANGELOG.md:7]: ah, ok... no this should be fine.

Comment (andrewrynhard) [reply CHANGELOG.md:7]: I usually have more than one commit in a back port, FWIW.

Comment (smira) [reply CHANGELOG.md:7]: yep, makes sense, I got stuck with 1 commit mindset, I'll do it better next time

**Pull request merged**